### ISSUE TITLE
Обеспечить доступ к hub.oscript.io по https

### ISSUE THREAD

**Issue by EvilBeaver** [EvilBeaver/oscript-library#13]
Возможно, потребуется доработка со стороны 1Script для валидации серверного сертификата

**Comment by dmpas** [comment]
let's encrypt запустили: http://habrahabr.ru/post/272253/
буду вечером на своих серверах пробовать

**Comment by EvilBeaver** [comment]
А у меня на hub nginx что-то мозг парит ((

**Comment by dmpas** [comment]
Тут всё глухо до сих пор?

**Comment by EvilBeaver** [comment]
Нет, но сильно ли тебе поможет self-signed https. Толку-то с него?

**Comment by dmpas** [comment]
Letsencrypt?
